dockerfiles/dbt.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.11-slim
+LABEL org.opencontainers.image.source=https://github.com/kestra-io/examples
+LABEL org.opencontainers.image.description="Image with the latest dbt-core Python package including all officially supported adapters."
+RUN pip install --no-cache-dir kestra dbt-bigquery 
```

Comment:
you should add the multiple pip on the same `RUN`, ex: `pip install --no-cache-dir dbt-snowflake dbt-redshift ...`
You are creating multiple docker layer right now that is useless

Reply:
I did it on purpose in case one of the installation steps would fail, easier to restart 😬 but agree, changed now